client/src/pages/Upgrade.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Divider from '@material-ui/core/Divider';
+import Header from '../components/Header';
+import CheckIcon from '@material-ui/icons/Check'
+
+import StripeCheckoutModal from '../components/stripe/StripeCheckoutModal';
+
+const useStyles = makeStyles((theme) => ({
+  '@global': {
+    ul: {
+      margin: 0,
+      padding: 0,
+      listStyle: 'none',
+    },
+  },
+
+  link: {
+    margin: theme.spacing(1, 1.5),
+  },
+
+  heroContent: {
+    padding: theme.spacing(8, 0, 6),
+  },
+
+  title: {
+    fontSize:40,
+    color:"secondary" 
+  },
+
+  card: {
+    width:350
+  },
+
+  cardHeader_premium: {
+    paddingTop:50,
+    marginBottom:13,
+    backgroundColor:theme.bgcolor,
+    color:theme.palette.purple.main
+  },
+
+  cardHeader_professional: {
+    paddingTop:50,
+    marginBottom:13,
+    backgroundColor:theme.bgcolor,
+    color:theme.palette.green.main
+  },
+
+  cardAction: {
+      marginBottom:35
+  },
+
+  cardButton: {
+    margin:"auto", 
+    width:150,
+    height:50,
+    '&:hover': {
+      background: theme.palette.primary.main,
+      color: theme.bgcolor
+   }
+  },
+
+  cardContent: {
+      height:170,
+      color:theme.palette.light.minorLight,
+      padding:'30px 40px 60px 40px',
+  },
+
+  checkIcon: {
+    color:theme.palette.primary.main,
+    fontSize:15,
+    marginRight:10
+  }
+}));
+
+const tiers = [
+  {
+    title: 'Standard',
+    price: 'Free',
+    description: [
+      'Unlimited event types',
+      'Group meetings',
+    ],
+    buttonText: 'Upgrade',
+    buttonVariant: 'outlined',
+  },
+  {
+    title: 'Premium',
+    price: '12',
+    description: [
+        'Unlimited event types',
+        'Group meetings',
+        '6 calendar connections'
+    ],
+    buttonText: 'Upgrade',
+    buttonVariant: 'outlined',
+  },
+];
+
+export default function Upgrade() {
+  const classes = useStyles();
+
+  const [openModal, setOpenModal] = useState(false)
+
+  const handleClickOpen = () => {
+    setOpenModal(true);
+  };
+
+  const handleClose = () => {
+    setOpenModal(false);
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
We can import `Fragment` and use `<Fragment>` instead of `<React.Fragment>`

Reply:
Or one convenient way of doing this can be `<></>` and I think we don't need to import `Fragment` in that way, only need to import `React`

Reply:
ohh ok good to know